openshift/openshift-ansible-contrib
# Add Extra CAs (custom post-provision action)

#### What does this PR do?
This PR enables adding extra CAs to the trust chain of each selected node.

#### How should this be manually tested?
1. Run provisioning.
2. Connect to an arbitrary node, try downloading some content from a domain that hasn't got its certificate stored in the trust chain of the node. Make sure it fails with a related error message.
3. Run the add-cas.yml playbook as follows:
```
ansible-playbook -i <inventory> openshift-ansible-contrib/playbooks/provisioning/openstack/custom-actions/add-cas.yml -e ca_files= <absolute path to ca file>
```
4. Make sure the playbook is run successfully.
5. Connect to the same node and try downloading the content again. It should now succeed.

#### Is there a relevant Issue open for this?
--

#### Who would you like to review this?
cc: @tomassedovic @bogdando @tzumainn  PTAL
